src/routes/blogs/olive-quant-ft/+page.svx

```diff
@@ -0,0 +1,196 @@
+      max_seq_len: 1024
+    dataloader_config:
+      batch_size: 8
+evaluators:
+  common_evaluator:
+    metrics:
+      - name: tinycodes_ppl
+        type: accuracy
+        sub_types:
+          - name: perplexity
+        data_config: tinycodes_ppl
+passes: &lbrace;&rbrace;
+auto_optimizer_config:
+  disable_auto_optimizer: true
+evaluator: common_evaluator
+host: local_system
+target: local_system
+output_dir: models/eval
+</code></pre>
+
+> **Note**: We define the same configurations for the other models but updated the `input_model`.
+
+We then executed the Olive configuration using:
+
+<pre><code>olive run perplexity-config.yaml</code></pre>
+
+## 📊 Results
+
+### Phi-3.5-Mini-Instruct
+
+The chart below shows the perplexity metrics for the:
+
+1. Different Quantization and Fine-tuning sequences (magenta) 
+1. Phi-3.5-Mini-Instruct base model (dashed green line), which is not quantized
+1. Phi-3.5-Mini-Instruct Fine-tuned model (solid green line), which is not quantized
+
+<div class="m-aut w60">
+<img src="./perplex-phi.png" alt="Perplexity metrics for Phi-3.5">
+
+The goal is for the quantized models to be as close to the fine-tuned model (solid green line) as possible. There are several takeaways:
+
+- Quantization does not have a significant impact on the model quality - as seen by the closeness of the perplexity scores for quantized models to the fine-tuned base model.
+- Quantizing *before* fine-tuning does give better results than quantizing after finetuning.
+- GPTQ provides better accuracy in this scenario than AWQ.
+
+### Llama-3.1-8B-Instruct
+
+The chart below shows the perplexity metrics for the:
+
+1. Different Quantization and Fine-tuning sequences (blue) 
+1. Llama-3.1-8B-Instruct base model (dashed green line), which is not quantized
+1. Llama-3.1-8B-Instruct Fine-tuned model (solid green line), which is not quantized
+
+<div class="m-aut w60">
+<img src="./perplex-llama.png" alt="Perplexity metrics for Llama-3.1-8B-Instruct">
+
+The goal is for the quantized models to be as close to the fine-tuned model (solid green line) as possible. There are several takeaways:
+
+- Quantization does not have a significant impact on the model quality - as seen by the closeness of the perplexity scores for quantized models to the fine-tuned base model.
+- Quantizing *before* fine-tuning does give better results than quantizing after finetuning.
+- GPTQ and AWQ give similar model quality results.
+
+## Conclusion
+
+In this blog post, we demonstrated how we utilised Olive to address common AI model optimisation queries. Our findings revealed that quantizing before fine-tuning significantly enhances model quality for both Phi-3.5-mini-instruct and Llama-3.1-8B-Instruct. These quantied variants closely match the quality of their full precision (FP32) counterparts, while requiring less memory and storage. This underscores the potential for on-device AI to deliver high-quality performance with a reduced resource footprint.
```

Comment:
significantly might be a bit strong given the data? For perplexity metrics, the difference doesn't appear big but based on how specialized the data is and how strict the metrics are, the difference is sometimes much more significant. But we don't have such a scenario in this blog.